src/mount/mount.ceph.c

```diff
@@ -268,6 +269,29 @@ static int parse_options(const char *data, struct ceph_mount_info *cmi)
 			/* ignore */
 		} else if (strcmp(data, "nofail") == 0) {
 			/* ignore */
+		} else if (strcmp(data, "fs") == 0) {
+			if (!value || !*value) {
+				fprintf(stderr, "mount option fs requires a value.\n");
+				return -EINVAL;
+			}
+			new_data = strdup("mds_namespace");
```

Comment:
I don't think `strdup` is necessary here.

Reply:
Yeah, will remove it.